examples/with-create-react-app/packages/eslint-config-custom/index.js

```diff
@@ -13,4 +13,9 @@ module.exports = {
       version: "detect",
     },
   },
+  parserOptions: {
+    babelOptions: {
+      presets: [require.resolve('next/babel')],
```

Comment:
We shouldn't need this one either, this example is for CRA and not Next (doesn't extend from next)